homeassistant/components/samsungtv/media_player.py

```diff
@@ -194,6 +199,11 @@ def is_volume_muted(self):
         """Boolean if volume is currently muted."""
         return self._muted
 
+    @property
+    def source_list(self):
+        """List of available input sources."""
+        return list(SOURCES.keys())
```

Comment:
Getting the key is the default.

```suggestion
        return list(SOURCES)
```